instrumentation/jedis/jedis-1.4/src/main/java/io/opentelemetry/auto/instrumentation/jedis/v1_4/JedisInstrumentation.java

```diff
@@ -44,7 +45,7 @@
 public final class JedisInstrumentation extends Instrumenter.Default {
 
   public JedisInstrumentation() {
-    super("jedis", "redis");
+    super("jedis", Constants.REDIS);
```

Comment:
this happens to match the constant, but is not really related to db_system (this is why I recommended to rename `Constants` to `DbSystem` above to make this more clear where to use it)
```suggestion
    super("jedis", "redis");
```